app/views/fairs/schedule.html.erb

```diff
@@ -15,8 +14,21 @@
     <% @entries.each do |entry| %>
       <tr>
         <td><%= entry.entry_name %></td>
-        <%= fields_for "entries[]", entry do |form| %>
-          <%= render partial: 'bulk_update_entry_timeslot', locals: {form: form, entry: entry} %>
+        <%= fields_for "entries[]", entry do |f| %>
+          <%= render partial: 'bulk_update_entry_timeslot', locals: {form: f, entry: entry} %>
+          <td>
+            <div class="field">
+              <%= f.select :location_id, Location.for_fair(entry.fair).map{ |m| [m.description, m.id] }%>
+            </div>
+          </td>
+          <td>
+            <%= f.fields_for "judge_assigns[]", entry do |ff| %>
```

Comment:
This feels like a hacky piece of code, but collection_check_boxes just returns everything mushed together in a single array, regardless of what entry it belongs to, and I can't figure out what entry each value belongs to.